content/docs/fairing/azure/_index.md

```diff
@@ -0,0 +1,121 @@
+
+```
+az aks show -g <RESOURCE_GROUP_NAME> -n <NAME>
+```
+
+Above command has output like:
+
+```
+  "servicePrincipalProfile": {
+    "clientId": "<id>"
+  },
+```
+
+The role you grant must have at least read and write permissions to ACR and Storage.
+
+To read more about how to grant the Service Principal access, please follow the documentation [here](https://docs.microsoft.com/en-us/azure/role-based-access-control/role-assignments-portal).
+
+## Setting up credentials as Kubernetes secrets
+
+Before running the notebook, you must first configure credentials so that the Python code running within the cluster can
+access the Azure resources required to train and deploy.
+
+One way to accomplish this is to first set the right environment variables and then run the commands below that use them.
+
+### Setting the environment variables variables
+
+```
+export AZ_CLIENT_ID=<Service Principal Client ID e.g. property `client_id` from ~/.azure/aksServicePrincipal.json>
+export AZ_CLIENT_SECRET=<Service Principal Client Secret e.g. property `client_secret` from ~/.azure/aksServicePrincipal.json>
+export AZ_TENANT_ID=<Tenant ID of the subscription e.g. `tenantId` from output of `az account show`>
+export AZ_SUBSCRIPTION_ID=<Subscription ID e.g. `id` from output of `az account show`>
+export TARGET_NAMESPACE=<target namespace e.g. kubeflow-anonymous>
+export ACR_NAME=<name of the configured ACR to which the Service Principal has access to e.g. kubeflowacr>
+```
+
+### Setting the secrets with kubectl
+
+```
+kubectl create secret generic -n ${TARGET_NAMESPACE} azcreds --from-literal=AZ_CLIENT_ID=${AZ_CLIENT_ID} --from-literal=AZ_CLIENT_SECRET=${AZ_CLIENT_SECRET} --from-literal=AZ_TENANT_ID=${AZ_TENANT_ID} --from-literal=AZ_SUBSCRIPTION_ID=${AZ_SUBSCRIPTION_ID}
+```
+
+```
+kubectl create secret docker-registry -n ${TARGET_NAMESPACE} acrcreds --docker-server=${ACR_NAME}.azurecr.io --docker-username=${AZ_CLIENT_ID} --docker-password=${AZ_CLIENT_SECRET}
+kubectl patch serviceaccount default-editor -n ${TARGET_NAMESPACE} -p "{\"imagePullSecrets\": [{\"name\": \"acrcreds\"}]}"
+```
+
+## Creating a Notebook Server in Kubeflow
+
+To create a Notebook Server, use your Web browser to access the Kubeflow Central Dashboard and select the `Notebook Servers` panel from the menu.
+
+First, select the target namespace in which you want to host the server. In the default Kubeflow installation, there should be a namespace `kubeflow-anonymous` available in the namespace drop-down menu.
+
+After the target namespace is selected, press the `NEW SERVER`-button and fill in the mandatory fields. The fields with default values can all be left as they are and do not have to be modified for the purpose of running the sample notebook.
+
+After launching the server, wait for the `CONNECT`-button to appear and press it to connect (it may take up to a minute for the server to ready for connection).
```

Comment:
Please remove the parenthesis from "(it may take up to a minute for the server to ready for connection)" and make it a separate sentence.